components/modal/index.en-US.md

```diff
@@ -0,0 +1,67 @@
+---
+type: Views
+category: Components
+english: Modal
+---
+
+Modal dialogs.
+
+## When to use
+
+When requiring users to interact with application, but don't want to jump to a new page to interrupt
```

Comment:
When requiring users to interact with application, but without jumping to a new page to interrupt